src/tests/ftest/io/dfuse_find_cmd.py

```diff
@@ -0,0 +1,215 @@
+                "DAOS is equal or faster running '{0}' tag".format(tag))
+
+    def _run_commands(self, test_path, samples, dir_trees):
+
+        profiler = general_utils.SimpleProfiler()
+        profiler.set_logger(self.log.info)
+
+        self.log.info("Sampling path: {0}".format(test_path))
+
+        for i in range(samples):
+            self.log.info(
+                "Running sample number {0} of {1}".format(i + 1, samples))
+            profiler.run(
+                self._run_cmd,
+                "all_files",
+                u"find {0} -name *.needle".format(test_path))
+
+            number = random.randrange(self._needles - 1)
+            file_name = "*_{:05d}.needle".format(number)
+            profiler.run(self._run_cmd, "same_suffix",
+                         u"find {0} -name {1}".format(test_path, file_name))
+
+            dir_tree = random.choice(dir_trees)
+            file_name, _ = dir_tree.get_probe()
+            profiler.run(self._run_cmd, "unique_file",
+                         u"find {0} -name {1}".format(test_path, file_name))
+
+        return profiler
+
+    def _setup_containers(self, dfs_path, cont_count):
+        mount_dirs = []
+        for count in range(cont_count):
+            self.add_container(self.pool)
+            cont_dir = "{}_daos_dfuse_{}".format(self.pool.uuid, count)
+            mount_dir = os.path.join(dfs_path, cont_dir)
+            self.log.info(
+                "Creating container Pool UUID: {0} Con UUID: {1}".format(
+                    self.pool, self.container))
+            self.start_dfuse(
+                self.hostlist_clients, self.pool, self.container, mount_dir)
+            self._dfuses.append(self.dfuse)
+            self._containers.append(self.container)
+            mount_dirs.append(mount_dir)
+
+        return mount_dirs
+
+    def _setup_challenger(self, dirs, challenger_path):
+        challenger_dirs = []
+
+        for dir in range(dirs):
+            prefix = "dir_{}_".format(dir)
+            tdir = tempfile.mkdtemp(dir=challenger_path, prefix=prefix)
+            challenger_dirs.append(tdir)
+
+        return challenger_dirs
+
+    def _crate_dir_trees(self, paths):
+        dir_trees = []
+
+        profiler = general_utils.SimpleProfiler()
+        profiler.set_logger(self.log.info)
+
+        for path in paths:
+            self.log.info("Populating: {0}".format(path))
+            dir_tree = DirTree(
+                path,
+                self._height,
+                self._subdirs_per_node,
+                self._files_per_node)
+            dir_tree.set_number_of_needles(self._needles)
+            tree_path = profiler.run(dir_tree.create, "create_dirtree")
+            dir_trees.append(dir_tree)
+            self.log.info("Dir tree created at: {0}".format(tree_path))
+
+        profiler.print_stats()
+        max_time, min_time, avg_time = profiler.get_stat("create_dirtree")
+        self.log.info(
+            "Max: {0} Min: {1}: Avg: {2} ".format(
```

Comment:
(pylint-logging-format-interpolation) Use % formatting in logging functions and pass the % parameters as arguments